docs/advanced-features/self-signed-certificates.md

```diff
@@ -0,0 +1,42 @@
+<!-- markdownlint-disable-next-line first-line-h1 -->
+To use a self-signed certificate with the `quilt3` API or CLI,
+the certificate must first be added to your Python environment.
```

Comment:
Need to explain that this procedure is useful when using `quilt3` with a client-to-site VPN.